site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,431 @@
+# Migrating from Ingress
+
+The Gateway API project is the successor to the Ingress API. However, it does
+not include the Ingress resource (the closest parallel is the HTTPRoute). As a
+result, a one-time conversion from your existing Ingress resources to the
+relevant Gateway API resources is necessary.
```

Comment:
We link to `Ingress` below, but since we're basically "introducing it" here we should have a link up here as well. Also a language nit:

```suggestion
The Gateway API project is the successor to the [Ingress][ing] API. However, it
does not include the `Ingress` resource (the closest parallel is the `HTTPRoute`
resource). As a result, a one-time conversion from your existing Ingress resources
to the relevant Gateway API resources is necessary.

[ing]:https://kubernetes.io/docs/concepts/services-networking/ingress/
```